core/conventional-commits/lib/recommend-version.js

```diff
@@ -26,8 +30,8 @@ function recommendVersion(pkg, type, { changelogPreset, rootPath, tagPrefix, pre
     options.tagPrefix = tagPrefix;
   }
 
-  const shouldBumpPrerelease = (releaseType, version) => {
-    if (!semver.prerelease(version)) {
+  const shouldBumpPrerelease = (releaseType, version, bumpPrerelease) => {
```

Comment:
Given this function's purpose is to dynamically determine whether or not to bump, it feels strange to invoke it with a ready made answer... I think let's instead refactor line 62 (what is currently line 66 in this diff)